management-app/src/main/resources/static/WEB-INF/jsp/createProduct.jsp

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE HTML>
```

Comment:
Why do you have one version of this JSP in `management-app/src/main/resources/static/WEB-INF/jsp` and another one in `management-app/src/main/webapp/WEB-INF/jsp`?